base/200-role.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: tekton-dashboard-info
+  namespace: tekton-pipelines
```

Comment:
The Dashboard supports installation in custom namespaces (other than `tekton-pipelines`) and handles this via its installer script which replaces `namespace: tekton-dashboard` with the expected value. See the script here https://github.com/tektoncd/dashboard/blob/97d983f043ea7a1f23d88d408c992f9f981bc789/scripts/installer#L167

Can you change the namespace here to `tekton-dashboard` so it will respect the install namespace setting? The same change would be needed in the other files too.